app/index.js

```diff
@@ -1,10 +1,12 @@
 import { setup } from './insights.js'
 import 'log-timestamp'
 import { createServer } from './server.js'
+import { startMessaging } from './messages/index.js'
 
 const init = async () => {
   const server = await createServer()
```

Comment:
Do we need this server if all this repo does is consume from service bus?  Can we just remove the Hapi stuff?

Reply:
Server has now been removed.